triton/client/utils.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
```

Comment:
This file has no copyright info and it is not clear where it is from.

Could you use https://github.com/pzelasko/kaldialign
to calculate the WER instead of reinventing the wheel.

We are alreadying using it in icefall.

Reply:
Done.